frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts

```diff
@@ -70,14 +62,14 @@ export const getDefaultUISchemaForPropertyName = (name) =>
 
 // Determine if a schema will produce an empty form field.
 export const hasNoFields = (jsonSchema: JSONSchema6): boolean => {
-  const type = getSchemaType(jsonSchema ?? {});
+  const type = getSchemaType(jsonSchema ?? {}) ?? '';
```

Comment:
Default to empty string when type cannot be guessed and add new case in switch to handle it. If the schema type cannot be determined by `getSchemaType`, the schema will not render any fields.